code/script.js

```diff
@@ -1,51 +1,312 @@
   } else if (sender === 'bot') {
+    //console.log('bot-message') //added console log
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
+        <img src="assets/bot-hotel.svg" alt="Bot" />
+            <div class="bubble bot-bubble">
+              <p class="p-bubble-msg">${message}</p>
         </div>
       </section>
     `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  } 
+    // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box and adds sound
   chat.scrollTop = chat.scrollHeight;
+  messageSound();
 }
 
-// Starts here
+
+// Conversations starts here
+
+// Question 1 - Bot asks 'What is your room number'?
+
 const greeting = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+      showMessage("Welcome to Hotel Vosje. What is your room number?", 'bot'),2000; 
+    };
+  
+
+// User answers with room number
+
+const handleFormInput = (event) => {        // at submit this function will be invoked
+  event.preventDefault();        // prevents website refresh at submit
+  const room = roomInput.value;        // input value will be stored in the const name
+  showMessage(`My room number is ${room}`, 'user');      // users answer 
+  roomInput.value = '';        // clearing room input setting it to an empty string
+
+  if (room === '') {       // this will prompt the user to answer correctly
+    showMessage(`Please, give me your room number.`, 'bot');
+  } else {
+    greetingLoop(room);
+  }
+}
+
+// The bot answers hello room #. This const will also be used as loop back selections if guest needs something else
+
+const greetingLoop = (room) => {
+  typing(); 
+  setTimeout(() => removeTyping(), 600)
+  showMessage(`Hello room ${room}`, 'bot');
+  setTimeout(() => helpSelection(), 600)
+}
+
+// Question 2 - Bot asks 'What do you need help with today?' and sends selection-buttons
+
+const helpSelection = () => {
+  showMessage('What can I help you with?', 'bot');
+  inputWrapper.innerHTML = `
+  <button id="amenities" type="sbumit" class="chat-btn">Amenities </button>
+  <button id="room-service" type="submit" class="chat-btn">Room service </button>
+  <button id="wake-up" type="submit" class="chat-btn">Wake-up call </button>
+  `
+  document.getElementById('amenities').addEventListener('click',() => {
+    showMessage(`I need amenities, please`, 'user')
+    setTimeout( ()=> {
+      preloader.style.display = "none";
```

Comment:
What is style.display doing? Why not only setTimeout? Maybe some comment here?

Reply:
oh i think this is actually an error and this "preloader.style.display ="none"" should not be there. it was from when we tried to make the bot-typing function works and it seems we forgot to remove it there :)